uiowa/uiowa
# [CTA] UIDS4 update

Resolves https://github.com/uiowa/uiowa/issues/8234. 

# How to test

```
ddev blt frontend && ddev blt ds --site=sandbox.uiowa.edu && ddev drush @sandbox.local uli /cta-testing
```
1. Review https://github.com/uiowa/uids/pull/946
2. Review this code
3. Compare against https://sandbox.prod.drupal.uiowa.edu/cta-testing
4. Check that the alignment styles were updated with the update hook
5. Create a new CTA and test out alignment style combinations
6. Test default site install for default content: `ddev blt di --site=default`

**Noticable changes and questions:**
- If using the CTA for the button only, you will notice that the inline version appears left aligned without content instead of centered.
- The centered option actually works now, previously it appears that it was just duplicating the inline style and perhaps not getting the correct styles.  This will mean that those that used the "Flex row center" option will have their previous CTAs updated into the centered option.  **_Should we just update those into the "inline" option instead?_**
